semantics/executable-spec/src/Control/Iterate/SetAlgebraInternal.hs

```diff
@@ -990,6 +1020,15 @@ compile (Intersect rel1 rel2) = (andPD (fst(compile rel1)) (fst(compile rel2)) (
 -- run materializes compiled code, only if it is not already data
 -- ===========================================================================
 
+testing :: Bool
+testing = False
```

Comment:
Would it makes sense to be able to run all of our tests, such as the property tests, in a way that catches these errors without having to manually change this variable?

Reply:
I have added some comments explaining what this is for. When we figure out how to pass this information, it will be a matter of changing this one line. Perhaps use the same mechanism Nick uses for assertions?

Reply:
We can change this easily when we decide on a test propogation strategy.